README.md

```diff
@@ -1,21 +1,25 @@
-## _Git Automation for Agile Development Teams_
+<img src="documentation/logo.png">
+
 <a href="https://travis-ci.org/Originate/git-town" alt="Build Status" target="_blank"><img src="https://travis-ci.org/Originate/git-town.svg?branch=master"></a>
 [![License](http://img.shields.io/:license-MIT-blue.svg?style=flat)](MIT-LICENSE)
 
-Git Town provides a number of additional Git commands that
-automate the typical high-level operations in
-[GitHub Flow](http://scottchacon.com/2011/08/31/github-flow.html)
-and other workflows.
+# _Git Automation for Agile Development Teams_
```

Comment:
I think this should go back to `##` over `#` so the title fits on a single line in Github.